### ISSUE TITLE
Windows XP Shared Folder stopped mounting with winrm>=1.3.5

### ISSUE THREAD

**Issue by jamusj** [mitchellh/vagrant#6839]
I'm seeing an issue where my Windows XP guest isn't mounting the shared folder starting in 1.7.3.

Using git bisect, I tracked it down to commit 31163da, which updated the winrm dependency from "~>0.1.0" to "~> 1.3.0".  Further testing show that winrm versions 1.3.0-1.3.4 work, while 1.3.5, 1.3.6, and 1.4.0 don't. 

On vagrant versions exhibiting the problem, it's failing silently.  From what I can tell, the mklink command is not being run.  I've modified the mklink invocation to log what happens:

```
cmd /c mklink /D "$MountPoint" "$VmProviderUncPath" > c:\mklink.out
```

I get a 0 byte file on winrm versions 1.3.5 and after, and the expected output for versions 1.3.4 and before.  

This is using the modern.ie virtualbox image, with minimal updates to add winrm and mklink support (http://www.emoxter.com/welcome-to-ghost/: "Needed software for XP" section, except for ssh).  

[Vagrantfile.txt](https://github.com/mitchellh/vagrant/files/83993/Vagrantfile.txt)

[vagrant-up.txt](https://github.com/mitchellh/vagrant/files/83994/vagrant-up.txt)

**Comment by jamusj** [comment]
I was able to create a simple ruby script using winrm that exhibits the problem, and opened https://github.com/WinRb/WinRM/issues/173 for it.

**Comment by jamusj** [comment]
If using the official binaries, then it depends on the version of winrm packaged if you see this or not.  On the Mac, 1.7.4 official binaries were packaged with winrm 1.3.3, so it won't exhibit the problem, while the version built from source will since it'll use 1.3.6.

**Comment by sneal** [comment]
Technically [Vagrant doesn't support Windows XP](https://github.com/mitchellh/vagrant/blob/b721eb62cfbfa93895d0d4cf019436ab6b1df05d/website/docs/source/v2/boxes/base.html.md#L182), but that's kind of a lame response for something that previously worked. On the other hand the WinRM change fixed issues for non-English supported Windows OSs.

**Comment by mwrock** [comment]
will be fixed with https://github.com/WinRb/WinRM/pull/182

**Comment by mwrock** [comment]
released fix in WinRM v1.6.1 https://github.com/WinRb/WinRM/blob/v1.6.1/changelog.md

**Comment by sneal** [comment]
@mwrock Thanks! Closing, the next Vagrant release will pull in the updated WinRM gem with the fix.
